backend/service/audit/audit.go

```diff
@@ -137,29 +139,44 @@ func (c *client) ReadEvents(ctx context.Context, start time.Time, end *time.Time
 	return c.storage.ReadEvents(ctx, start, end)
 }
 
-// This should be called via `go` in order to avoid blocking main exectuion.
-func (c *client) poll() {
-	readAndFanout := func(ctx context.Context) {
-		// TODO(maybe): Backpressure on continued failure.
-		events, err := c.storage.UnsentEvents(ctx)
-		if err != nil {
-			return
-		}
+func (c *client) readAndFanout(ctx context.Context) {
+	// TODO(maybe): Backpressure on continued failure.
+
+	c.sinkWriterScope.Counter("events_fetch").Inc(1)
```

Comment:
i was going to suggest an `events_fetch` success & error, but with gauge and counter i think it will make it pretty clear if we are receiving anything at all.

Reply:
i added error just in case